examples/n_crassa_WT.yml

```diff
@@ -10,6 +10,8 @@ sra_ids:
 
 hicpro_restriction_sites: "A^AGCTT"
 
+hicpro_ligation_site: "AAGCTAGCTT"
```

Comment:
Is it the appropriate ligation site for this N. crassa dataset?

Reply:
Yes, it is the appropriate ligation site. Thank you @pierrepo